docs/src/ci-intro-js.md

```diff
@@ -92,16 +94,15 @@ Locally opening the report will not work as expected as you need a web server in
 npx playwright show-report name-of-my-extracted-playwright-report
 ```
 
-<img width="752" alt="Viewing the HTML Report" src="https://user-images.githubusercontent.com/13063165/183437645-b47dd175-2e07-4ecc-a469-27d5b150b7ed.png" />
+<img width="1404" alt="Playwright HTML Report" src="https://user-images.githubusercontent.com/13063165/212745273-c19487d2-bc5e-483f-9f67-f9c9e5413ff4.png" />
 
 To learn more about reports check out our detailed guide on [HTML Reporter](/test-reporters.md#html-reporter)
 
 ### Viewing the Trace
 
 Once you have served the report using `npx playwright show-report`, click on the trace icon next to the test's file name as seen in the image above. You can then view the trace of your tests and inspect each action to try to find out why the tests are failing.
 
-
-<img width="1907" alt="Viewing the Trace" src="https://user-images.githubusercontent.com/13063165/183879653-d442e6b4-14f5-4d0e-99f3-9ba19f82c7cf.png"/>
+<img width="2005" alt="Playwright Trace Viewer" src="https://user-images.githubusercontent.com/13063165/212746021-c3c5dabb-c5dd-4e4e-adf5-54b225d78dc8.png" />
```

Comment:
% internal selector thing which should be fixed on next.

Reply:
updated